_posts/2016/2016-09-28-rest-api-test-framework.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: "How we test our API"
+date: "2016-09-28 11:11:11"
+image: blog-header/heartbeat.jpg
+categories: tech-stories
+authors: ["Ulf B."]
+---
+
+A few years ago, at the time when the development of the ePages REST API started,
+ePages 6 was just a huge monolith written in Perl and already hard to handle.
+Since the monolith should not be inflated even more and in order to be able to use state-of-the-art frameworks,
+the REST API was started as a separate service implemented in Java.
+
+This service communicates with the monolith via REST and SOAP in order to operate on the data we won't to expose to the public.
+All the API related business logic like right management and rate limits are implemented in Java such that the API's we had to implement in Perl could
```

Comment:
geht auch sowas wie "in order to keep the API's we implemented in Perl quiete simple"